Demo/WhisperDemo/WhisperDemo/ViewController.swift

```diff
@@ -3,13 +3,13 @@ import Whisper
 
 class ViewController: UIViewController {
 
-  lazy var scrollView: UIScrollView = UIScrollView()
+  lazy var scrollView = UIScrollView()
 
-  lazy var titleLabel: UILabel = {
+  lazy var titleLabel: UILabel = { [unowned self] in
```

Comment:
Why did you add `[unowned self]` here?
`self` is not referenced inside of the closure so I think we can go without it.